tango.pc.cmake

```diff
@@ -2,10 +2,11 @@ prefix = @CMAKE_INSTALL_PREFIX@
 exec_prefix = ${prefix}
 includedir = ${prefix}/include
 libdir = ${prefix}/@CMAKE_INSTALL_LIBDIR@
+devices_dir = ${prefix}/@CMAKE_INSTALL_LIBDIR@/tango
```

Comment:
If I understood correctly, this new variable will be used to know the default installation directory for device servers.
If this is the case, I would suggest to call it dserver_default_dir or ds_default_dir or something similar.
It would probably make sense to install them by default under ${prefix}/@CMAKE_INSTALL_BINDIR@ or ${prefix}/@CMAKE_INSTALL_BINDIR@/tango
But unless I misunderstood the purpose of this variable (very probable), I would suggest to avoid to install the device servers anywhere under ${prefix}/@CMAKE_INSTALL_**LIBDIR**@